src/modules/functionalities.test.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @jest-environment jsdom
+ */
+import { displayTasks, removeTask } from './functionalities.js';
+import Task from './task.js';
+
+const Tasks = new Task();
+
+describe('Add Task and local storage test', () => {
+  window.localStorage = Storage.prototype;
+  test('Add new Task', () => {
+    const task = {
+      index: 0,
+      description: 'Task 1',
+      completed: false,
+    };
+    Tasks.addTask(task);
+    document.body.innerHTML = '<ul class="nav--list"></ul>';
+    const listContainer = document.querySelector('.nav--list');
```

Comment:
Please that your project has a test file ([..].test.js) for a file containing the add item and delete item functions that would be tested. Kindly move your test to `test.js` file according to the project requirement.

Reply:
Dear Mr Micheaol,
Sorry, but I didn't understand what is the mistake here as the project requirement I create a [..].test.js, My file that contains the add and deletes functionality is the functionalities.js file so as I could understand the [..] mean my file name so that's why my test file is named functionalities.test.js. 
https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/tests_1.md

Reply:
@Mo7amedSMahdi ,

Sorry for the late reply.  Please see the example below:
 
   `........src`
   `then inside your src folder`
    `you should have test.js file`

I hope this helps